code/citadel/cit_virology.dm

```diff
@@ -0,0 +1,244 @@
+	return
+
+/datum/symptom/oversensitive/End(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	to_chat(M,"<span class='notice'>You feel your loins calm down.</span>")
+	return
+
+
+/*
+//////////////////////////////////////
+Desensitize
+
+	Less Noticable.
+	Slightly Less Resistant
+	Progresses faster.
+	More transmittable
+	Intense Level.
+	Low severity
+Bonus
+	Decreases arousal.
+	Makes the user a unable to feel physical sensations properly.
+//////////////////////////////////////
+*/
+
+/datum/symptom/desensitize
+
+	name = "Desensitize"
+	stealth = 2
+	resistance = -1
+	stage_speed = 2
+	transmittable = 2
+	level = 4
+	severity = 1
+	var/uncomfortable_thoughts = list("You feel a bit uncomfortable.","You feel like your clothes don't fit properly.","Your skin feels oddly stiff.")
+	var/discouraged_thoughts = list("You feel numbed.","Your skin feels like soft plastic.","You can't feel any pain.","You feel like nothing is real anymore.")
+
+/datum/symptom/desensitize/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/M = A.affected_mob
+
+		switch(A.stage)
+			if(1)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(2)
+				M.adjustArousalLoss(-1)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(3)
+				M.adjustArousalLoss(-2)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(4)
+				to_chat(M,"<span class='notice'>[pick(discouraged_thoughts)]</span>")
+				M.adjustArousalLoss(-3)
+			else
+				to_chat(M,"<span class='notice'>[pick(discouraged_thoughts)]</span>")
+				M.adjustArousalLoss(-6)
+	return
+
+/datum/symptom/desensitize/Start(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	to_chat(M,"<span class='notice'>You feel strangely uncomfortable.</span>")
+	return
+
+/datum/symptom/desensitize/End(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	to_chat(M,"<span class='notice'>You feel less numb again.</span>")
+	return
+
+
+/*
+//////////////////////////////////////
+Extendong
+
+	Obviously noticable
+	Increases Resistance
+	Slightly faster (quickshot, *laughtrack noises*)
+	Lowers transmission
+	Pretty low Level.
+	Not severe
+Bonus
+	Contrary to popular belief, this makes all genitals increase in size
+	The bane of tiny traps
+//////////////////////////////////////
+*/
+
+/datum/symptom/macro_genitals
+
+	name = "Extendong"
+	stealth = -3
+	resistance = 2
+	stage_speed = 1
+	transmittable = -2
+	level = 2
+	severity = 1
+	var/big_thoughts = list("You feel a bit stiff.","Your loins itch.","You feel like you don't want to wear clothing.")
+
+
+/datum/symptom/macro_genitals/proc/grow_genitalia(mob/living/carbon/human/M)
+	if(M.has_penis())
+		var/obj/item/organ/genital/penis/O = M.getorganslot("penis")
+		if(O.length < COCK_SIZE_MAX) //Couldn't find a macro or anything for the max size of a dong, so hardcoded, sadly.
```

Comment:
yeah, I think I forgot to remove the comment after finding the constant. When I wrote the comment, I had just put "20" there XD.